bucklescript/test/StringTest.re

```diff
@@ -28,6 +28,50 @@ let suite =
       });
     });
 
+    describe("firstIndexOf", () => {
+      test("returns some index of the first matching character", () => {
+        expect(
+          firstIndexOf("hello", ~char='h') |> Option.unwrap(~default=9999),
+        )
+        |> toEqual(Eq.int, 0)
```

Comment:
There is an Eq.option function which is used like so:
```
|> toEqual(Eq.(option(int)), Some(0))
```

That means you won't need the `Option.unwrap` in your tests.

Reply:
Switched in: d497d57